gapic/lang/common.yaml

```diff
@@ -5,6 +5,7 @@ common:
   package_dependencies_yaml: ${REPOROOT}/googleapis/gapic/packaging/dependencies.yaml
   package_defaults_yaml: ${REPOROOT}/googleapis/gapic/packaging/api_defaults.yaml
   common_protos_yaml: ${REPOROOT}/googleapis/gapic/packaging/common_protos.yaml
+  organization_name: google-cloud
```

Comment:
I think I would prefer to put `google-cloud` in all of the artman yaml files - it's something we'll have to add everywhere later anyway as we expand to other organizations, which will overwhelm google-cloud in the number of APIs.